payment_environment/models/payment_acquirer.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Camptocamp SA
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+
+from odoo import models
+
+
+class PaymentAcquirer(models.Model):
+    _name = "payment.acquirer"
+    _inherit = [
+        "payment.acquirer",
+        "server.env.techname.mixin",
```

Comment:
you can save a line here tnx to https://github.com/OCA/server-env/pull/75

Reply:
Indeed :) but I'll wait until it's merged